packages/editor/src/utils/custom-functions/generatePythonCFCode.ts

```diff
@@ -32,8 +32,10 @@ function injectNamespace(script: string, namespace: string) {
   //    (and don't want to do it in the general case in the original regex, because
   //     a regex with a "g" flag becomes a state-ful object that needs to be reset -- see
   //     https://stackoverflow.com/questions/11477415 for more info)
+  const regex = new RegExp(PythonCFSnippetRegex, 'gm');
+
   return script.replace(
-    new RegExp(PythonCFSnippetRegex, 'gm'),
+    regex,
```

Comment:
why was this changed?

Reply:
Just to move the regex to a constant, where it can have a comment next to it.  And so that the call itself looks a tad simpler.